samples/azure-quantum/simulators/SparseSimulator.ipynb

```diff
@@ -0,0 +1,2274 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "acquired-exhibit",
+   "metadata": {},
+   "source": [
+    "# Sparse Simulator\n",
+    "\n",
+    "The sparse simulator is a simulator that utilizes a sparse representation of quantum state vectors, as opposed to the full state simulator. This feature allows the sparse simulator to minimize the memory footprint used to represent quantum states that are sparse in the computational basis, thus enabling simulations over a larger number of qubits. The sparse simulator is efficient for a class of quantum algorithms with a smaller number of states in superposition. This simulator is an unlocker for users to explore larger applications than what can be explored using the full state simulator alone. For more information about the sparse simulator see [this paper](https://arxiv.org/abs/2105.01533).\n",
```

Comment:
A few comments on this cell:

- Need something higher-level as an intro, since portal gallery samples may be discoverable by users new to quantum. The sample on the whole need not be beginner friendly, but the introduction should at least clearly set the context and expectations, and redirect to other resources if a user wants to exit to other learning paths.
- > The sparse simulator is efficient for a class of quantum algorithms with a smaller number of states in superposition.
  
  This isn't terribly precise as written — it would be more correct, I think, to note that the sparse simulator is efficient for representing quantum states that are sparse in the computational basis.
- > This simulator is an unlocker for users to explore larger applications than what can be explored using the full state simulator alone.

  This isn't quite gramatically correct, and is a bit tech-jargony. I would suggest something along these lines instead:

  > This simulator is useful for exploring larger applications than can be represented using dense simulation alone.

- > For more information about the sparse simulator see [this paper](https://arxiv.org/abs/2105.01533).

  For both accessibility and SEO, links should be descriptive. Please avoid "this paper" as link text. Here, I would suggest:
  
  > For more information about the sparse simulator, please see [Jaques and Häner (arXiv:2105.01533)](https://arxiv.org/abs/2105.01533).

Reply:
Updating.